slides/README.md

```diff
@@ -1,6 +1,6 @@
 # Workshop Slides
 
-Directory for slides of the [Simulation-based Inference workshop](mlcolab.org/sbi-workshop). It contains an example for `jupyter` slides with interactive elements. We strive to use these text-based slides as it makes further development simpler! 
+Directory for slides of the [Simulation-based Inference workshop](https://mlcolab.org/sbi-workshop). It contains an example for `jupyter` slides with interactive elements. We strive to use these text-based slides as it makes further development simpler! 
```

Comment:
Thanks for noticing the dead URL! Actually, it seems even with the `https://`, the URL is dead. This should point to the correct website: 

```suggestion
Directory for slides of the [Simulation-based Inference workshop](https://mlcolab.org/resources/simulation-based-inference-for-scientific-discovery). It contains an example for `jupyter` slides with interactive elements. We strive to use these text-based slides as it makes further development simpler! 
```

Reply:
Thank you @sethaxen for the correction.